modules/op-installing-pipelines-as-code-cli.adoc

```diff
@@ -7,21 +7,21 @@
 = Installing {pac} CLI
 
 [role="_abstract"]
-Cluster administrators can use the `tkn-pac` and `opc` CLI tools on local machines or as containers for testing. The `tkn-pac` and `opc` CLI tools are installed automatically when you install the `tkn` CLI for {pipelines-title}.
+Cluster administrators can use the `tkn pac` and `opc` CLI tools on local machines or as containers for testing. The `tkn pac` and `opc` CLI tools are installed automatically when you install the `tkn` CLI for {pipelines-title}.
```

Comment:
Do we need to use `opc` or `opc pac` :thinking: 
cc @chmouel

Reply:
i am not sure if opc is GA so we may have to keep tkn for now cc 
@vdemeester

Reply:
This is not resolved... it should say `opc pac`